frontend/libs/studio-components/src/components/StudioAvatar/StudioAvatar.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { StudioAvatar, type StudioAvatarProps } from './StudioAvatar';
+
+describe('StudioAvatar', () => {
+  afterEach(() => jest.clearAllMocks());
+
+  it('should render the default PersonCircleIcon when imageDetails is not provided', () => {
+    renderStudioAvatar();
+
+    const iconElement = screen.getByRole('img', { hidden: true });
+    expect(iconElement).toBeInTheDocument();
+    expect(iconElement).toHaveClass('avatar');
```

Comment:
I suggest that we remove the `.toHaveClass` expectation. The reason for this is that it does not give us that much value to know. We also have a separate issue to analyze which tool to use for visual testing, ex. Percy.